src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/registration/display_vetting_types.html.twig

```diff
@@ -52,5 +38,19 @@
               </div>
           </div>
         {% endif %}
+        <div class="col-xs-12 col-md-3">
+            <div class="thumbnail remote-vetting-selector">
```

Comment:
I think changing the css selector to vetting-selector would be more fitting. That way all other vetting types would also apply.